### ISSUE TITLE
Add new Drawing field: Artist's Reason for Migration

### ISSUE THREAD

**Issue by krissy** [empowerhack/DrawMyLife-Service#127]
## What

We need to add a new field to Drawings to give a high-level background on why the artist is migrating.
## How

Add a new select field to the Drawings model. Options TBC as per considerations below.
## Considerations

What are potential options for this field?
## To Do

@CathMollie to confirm options, consulting with Dama + @smurfyrach

**Comment by CathMollie** [comment]
Following call with @smurfyrach 19 Oct:
- New field, tagged approach
- Not mandatory as not all field workers will be comfortable collecting this
- May not share this data openly. Part of HDX/partner discussions.
- Worth enabling for pilot to we can look for common themes.

**Comment by tanyapowell** [comment]
@krissy @CathMollie I'm getting a bit concerned that the form is getting too long. Is it time for the entire form to be re-evaluated?

**Comment by CathMollie** [comment]
@tanyapowell Agree, though concerned we're losing time on gathering data in Serbia. Perhaps we opt to remove this for now and instead 'test' the idea of including it as part of next pilot. @krissy? I'll add this question our living doc for next pilot discussion guide/usability test:
https://docs.google.com/document/d/1RciZk4ZeFrqh9GOJcsFmpNYRRAg41fub78XSWQ80ZJ0/edit

**Comment by krissy** [comment]
@CathMollie @tanyapowell I agree this can be lowered in priority compared to other tasks and shouldn't block the pilot. Will remove the High Priority tag for now; we can always add it back in if a reason comes up.
